src/backend/src/routes/users.routes.ts

```diff
@@ -55,6 +55,7 @@ userRouter.post(
   UsersController.getManyUserTasks
 );
 userRouter.get('/:userId/notifications', UsersController.getUserUnreadNotifications);
+userRouter.get('/:userId/announcements', UsersController.getUserUnreadAnnouncements);
```

Comment:
i dont think we should be passing in userId to any of these routes. I could be confused. Message me on slack what userId actually represents here